### PR TITLE
Added info about FA1.2 version of stablecoin to Readme

### PR DESCRIPTION
**Pull request by dcastro**
## Description

In #114 we introduced an alternative version of stablecoin, implementing FA1.2 instead of FA2.

This PR adds info about this contract, as well as tables showing the gas/transaction costs and instructions on how to recreate these tables to the README.

It also modifies the nettests introduced in #114 to facilitate measuring and comparing gas/transaction costs.

#### Related changes (conditional)

- Tests
  - [x] If I added new functionality, I added tests covering it.
  - [x] If I fixed a bug, I added a regression test to prevent the bug from
        silently reappearing again.

- Documentation
  - [x] I checked whether I should update the docs and did so if necessary:
    - [README](../tree/master/README.md)
    - Haddock
  - [x] I updated [changelog](../tree/master/ChangeLog.md) unless I am sure my changes are
        not essential.

#### Stylistic guide (mandatory)

- [x] My commits comply with [the policy used in Serokell](https://www.notion.so/serokell/Where-and-how-to-commit-your-work-58f8973a4b3142c8abbd2e6fd5b3a08e).
- [x] My code complies with the [style guide](../tree/master/docs/code-style.md).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,6 +32,38 @@ It allows us to use features of `morley` (such as testing engine) with this cont
 Tests are implemented in the same Haskell package.
 Please refer to the [`haskell/`](/haskell/) directory for details.
 
+## Gas / Transaction costs
+
+The tables below show the gas and transaction costs of both versions (FA1.2 and FA2) of
+the stablecoin contract v1.3.0 in both Carthagenet and Delphinet.
+
```
Comment (gromakovsky) [inline README.md:39]: I would also provide links to better-call.dev or at least addresses of the contracts that were used for these experiments, maybe some people will want to see details (e. g. which transactions were made or how much storage was consumed).

File: README.md [hunk 2]
```diff
@@ -30,6 +32,38 @@ It allows us to use features of `morley` (such as testing engine) with this cont
 Tests are implemented in the same Haskell package.
 Please refer to the [`haskell/`](/haskell/) directory for details.
 
+## Gas / Transaction costs
+
+The tables below show the gas and transaction costs of both versions (FA1.2 and FA2) of
+the stablecoin contract v1.3.0 in both Carthagenet and Delphinet.
```
Comment (gromakovsky) [inline README.md:38]:
```suggestion
the stablecoin contract [v1.3.0](https://github.com/tqtezos/stablecoin/releases/tag/v1.3.0) in both Carthagenet and Delphinet.
```

**Review by gromakovsky — APPROVED** [review]
LGTM, only minor suggestions.

File: README.md (hunk 1 above)
Comment (dcastro) [reply README.md:39]: Good idea, done!

**Pull request merged**